opennft/matlab/nfbPreprVol/iGlmVol.m

```diff
@@ -86,9 +86,11 @@
 % statistical image masking, as set by SPM structure, see
 % setupProcParams.m
 recTh = recTh + spmMaskTh(n) * spmMaskTh(n); % cumulative sum of squares
-spmActVox = find(sigma2n > recTh);
+spmActVox = sigma2n > recTh;
 
 % intersect of iGLM and SPM indexes
-idxActVox.pos = intersect(iglmActVox.pos, spmActVox);
-idxActVox.neg = intersect(iglmActVox.neg, spmActVox);
+tn.pos = tn.pos .* spmActVox; 
+tn.neg = tn.neg .* spmActVox; 
+idxActVox.pos = intersect(iglmActVox.pos, find(spmActVox));
+idxActVox.neg = intersect(iglmActVox.neg, find(spmActVox));
```

Comment:
Double "find" is here. Maybe we could call "find" function once?

Reply:
Done in [commit 753baf7](https://github.com/OpenNFT/OpenNFT/pull/63/commits/753baf752cb5b0d47f313affbe3b42a07bdac680)